glean-core/src/event_database/mod.rs

```diff
@@ -21,7 +21,7 @@ use crate::Glean;
 use crate::Result;
 
 /// Represents the data for a single event.
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
```

Comment:
What is `PartialEq` useful for? (trying to learn a bit more)

Reply:
[PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html) give us `==` comparison, and [Eq](https://doc.rust-lang.org/std/cmp/trait.Eq.html) gives us that it's also reflexive (`a == a`).
This is required for the `assert_eq` test below.